<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew_2">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionModel1"/>
     <addaction name="actionModel2"/>
     <addaction name="actionModel3"/>
    </widget>
    <addaction name="menuNew_2"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_sql"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCoefficient"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCoefficient">
   <property name="text">
    <string>Coefficient</string>
   </property>
  </action>
  <action name="actionOpen_sql">
   <property name="text">
    <string>Open(sql)</string>
   </property>
  </action>
  <action name="actionModel1">
   <property name="text">
    <string>Model1</string>
   </property>
  </action>
  <action name="actionModel2">
   <property name="text">
    <string>Model2</string>
   </property>
  </action>
  <action name="actionModel3">
   <property name="text">
    <string>Model3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
